client/dashboard/me/security-two-step-auth/main-page/security-keys/index.tsx

```diff
@@ -0,0 +1,176 @@
+import { securityKeysQuery, deleteSecurityKeyMutation } from '@automattic/api-queries';
+import { useQuery, useMutation } from '@tanstack/react-query';
+import {
+	__experimentalConfirmDialog as ConfirmDialog,
+	Button,
+	Card,
+	CardHeader,
+	CardBody,
+	Icon,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { DataViews } from '@wordpress/dataviews';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { closeSmall } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import { useState } from 'react';
+import InlineSupportLink from '../../../../components/inline-support-link';
+import { SectionHeader } from '../../../../components/section-header';
+import { isWebAuthnSupported } from '../../hooks/use-register-security-key';
+import RegisterKey from './register-key';
+import type { UserSecurityKeys } from '@automattic/api-core';
+
+type SecurityKeyRegistration = UserSecurityKeys[ 'registrations' ][ number ];
+
+const fields = [
+	{
+		id: 'name',
+		label: __( 'Name' ),
+		getValue: ( { item }: { item: SecurityKeyRegistration } ) => item.name,
+	},
+];
+
+const view = {
+	fields: [],
+	type: 'list' as const,
+	titleField: 'name',
+};
+
+const SecurityKeysList = ( {
+	data,
+	isLoading,
+}: {
+	data: SecurityKeyRegistration[];
+	isLoading: boolean;
+} ) => {
+	const { mutate: deleteSecurityKey } = useMutation( deleteSecurityKeyMutation() );
+
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+
+	const [ selectedKeyToRemove, setSelectedKeyToRemove ] =
+		useState< SecurityKeyRegistration | null >( null );
+
+	const handleRemove = () => {
+		setSelectedKeyToRemove( null );
+		if ( selectedKeyToRemove ) {
+			deleteSecurityKey(
+				{ credential_id: selectedKeyToRemove.id },
+				{
+					onSuccess: () => {
+						createSuccessNotice( __( 'Security key removed successfully!' ), {
+							type: 'snackbar',
+						} );
+					},
+					onError: () => {
+						createErrorNotice( __( 'Failed to remove security key.' ), {
+							type: 'snackbar',
+						} );
+					},
+				}
+			);
+		}
+	};
+
+	return (
+		<>
+			<DataViews< SecurityKeyRegistration >
+				data={ data }
+				fields={ fields }
+				view={ view }
+				onChangeView={ () => {} }
+				getItemId={ ( item ) => item.id }
+				paginationInfo={ { totalItems: data.length, totalPages: 1 } }
+				defaultLayouts={ { list: {} } }
+				empty={ __( 'No security keys registered' ) }
```

Comment:
I know the default empty state message doesn't end in a period. But we use periods everywhere except headings.

```suggestion
				empty={ __( 'No security keys registered.' ) }
```